### ISSUE TITLE
Multiple snapshots for same test

### ISSUE THREAD

**Issue by DanDeMicco** [horvay/jasmine-snapshot#2]
I am using karma & jasmine to test some web components. I am trying to add snapshot testing via jasmine-snapshot. I was able to get a snapshot test working for the most part, but am running into a slight issue when running multiple browsers. I am running the tests in both firefox & chrome, and since web components are not natively supported in firefox, I am having to use a polyfill for the shadow dom. Unfortunately, this causes chrome & firefox to have a slightly different HTML structure (there are some additional classes added on firefox, which are not there in chrome, for the shadow dom polyfill).

Is there some way to allow multiple snapshots for a test? This actually sparked a workaround in my head, to sniff the user agent and add the snapshot in a conditional statement.

Also, some additional documentation on registering the snapshots in the README would be great.

Thanks for putting together this library btw!

**Comment by horvay** [comment]
Yes, I could use more documentation around the registering part. Thanks for bringing that up.

As for the problem between browsers, that has been the bane of my life with this library. I ended up just doing KeyExceptionList.push("class") for those test which I know is not a great solution.

I would indeed be nice to be able to conditionally add snapshots based on the runner.

**Comment by DanDeMicco** [comment]
Cool, thanks for adding some more documentation on the register snapshots. I still think it is a bit confusing. For example, when I read this 

> Note you can only have one registered snapshot object at a time at this point.

I would think that means I can only call registerSnapshots only once, and you have to have one giant object.  However, I was able to register once per describe with separate objects.

Also, I got around the issue by adding a conditional statement which will add the correct snapshot string depending on which browser.

**Comment by horvay** [comment]
Thanks about the documentation. I gave it another go.

Could you send me a quick sample of the conditional statement that detects the browser?

**Comment by DanDeMicco** [comment]
Yeah, it makes a bit more sense now. I would say create a new section called 'Registering Snapshots'  and put the info in there (maybe slimmed down copy of the example, with some comments).

As for what I did, I am just testing FF and chrome. I created a file 'util/browsers.js' with:

```javascript
import bowser from "bowser"
const isFirefox = bowser.name === "Firefox"
export default isFirefox
```

And in my test
```javascript
import isFirefox from "./util/browser"

const ffSnapshot =  "sample snapshotting basic examples does a basic html snapshot 1": `{ "div": { "_class": "ribbit frogs firefox", "_id": "freddy", "span": "hails yeahs" }}`
const chromeSnapshot =  "sample snapshotting basic examples does a basic html snapshot 1": `{ "div": { "_class": "ribbit frogs chrome", "_id": "freddy", "span": "hails yeahs" }}`

const snapshotObj = isFirefox ? ffSnapshot : chromeSnapshot

registerSnapshots(snapshotObj, "my test suite")
```
